scss/civicrm/full-text-search/_full-text-search.scss

```diff
@@ -0,0 +1,19 @@
+.CRM_Contact_Form_Search_Custom .crm-search-form-block table.form-layout-compressed td {
+  padding: 2px 0;
+  width: auto;
+
+  label {
+    margin: 0;
+  }
+
+  &.label {
+    font-weight: bold;
+    padding: 4px 20px;
+    vertical-align: middle;
+  }
+
+  & > input.crm-form-submit {
+    margin-left: 20px !important;
+    padding: $crm-btn-padding-small-vertical $crm-btn-padding-base-horizontal;
```

Comment:
Can @extend .btn-sm work here is small padding is needed? left and right padding seems excessive otherwise.